certify-service/src/main/java/io/mosip/certify/config/SecurityConfig.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package io.mosip.certify.config;
+
+import io.mosip.certify.core.config.LocalAuthenticationEntryPoint;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityCustomizer;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.csrf.CookieCsrfTokenRepository;
+import org.springframework.security.web.csrf.CsrfTokenRepository;
+
+
+import java.util.List;
+import java.util.Map;
+
+@Slf4j
+@Configuration
+@EnableWebSecurity
+@EnableMethodSecurity
+@Profile(value = {"!test"})
+public class SecurityConfig {
+
+    @Autowired
+    private LocalAuthenticationEntryPoint localAuthenticationEntryPoint;
+
+    @Value("${server.servlet.path}")
+    private String servletPath;
+
+    @Value("#{${mosip.certify.security.auth.post-urls}}")
+    private Map<String, List<String>> securePostUrls;
+
+    @Value("#{${mosip.certify.security.auth.put-urls}}")
+    private Map<String, List<String>> securePutUrls;
+
+    @Value("#{${mosip.certify.security.auth.get-urls}}")
+    private Map<String, List<String>> secureGetUrls;
+
+    @Value("${mosip.esignet.vci.authn.jwk-set-uri}")
```

Comment:
Why is this still pointing to mosip.esignet ?

Reply:
we are using esignet instance to get the public jwk for access token verification. So left it as is since this is a esignet reference

Reply:
Marking this resolved, since this comment is taken care